app/src/main/res/layout-land/topic_info_fragment.xml

```diff
@@ -26,58 +26,58 @@
 
       <TextView
         android:id="@+id/topic_name_text_view"
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="72dp"
         android:layout_marginTop="8dp"
         android:layout_marginEnd="72dp"
         android:fontFamily="sans-serif"
         android:text="@{viewModel.topic.name}"
         android:textColor="@color/oppiaPrimaryText"
-        android:textDirection="locale"
         android:textSize="24sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"/>
```

Comment:
Nit: space before `/>`

Reply:
Basically use reformatting shortcut in all files before merging.

Reply:
Thanks @rt4914. Done reformatting here and elsewhere.